src/main/java/jpa/domain/Member.java

```diff
@@ -0,0 +1,52 @@
+package jpa.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import jpa.infrastructure.jpa.BaseEntity;
+
+/**
+ * @author : leesangbae
+ * @project : jpa
+ * @since : 2020-12-15
+ */
+
+@Entity
+public class Member extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String email;
+
+    private String password;
+
+    private Integer age;
```

Comment:
```suggestion
    private int age;
```

테이블이 nullable이긴 하지만 코드에서는 null을 최소화해주는게 좋겠네요